src/components/common/Navbar.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+export type UserEmailType = {
```

Comment:
```suggestion
export type NavbarProps = {
```
Propsのtype名は`<Component name>Props`に統一しましょう！

Reply:
@HidemichiShimura 
確認ありがとうございます！
Button（`type.d.ts`）や`Input.tsx`のpropsのtype名は`<Component name>Type`になってますが、そのままでも問題ないですか？
そのままでもいい場合、変更しなくても大丈夫な理由を教えてもらえると助かります！

Reply:
@yurihoka 
そこも`<Component name>Props`に統一した方がいいですね！
対応お願いします

Reply:
@HidemichiShimura 
一旦NavbarのPropsのType名だけ変更したので、確認お願いします！
他のPropsのType名は別Issueで対応します！